bin/ch/Debugger.h

```diff
@@ -5,13 +5,12 @@
 #pragma once
 
 #ifdef _WIN32
-static const WCHAR controllerScript[] = {
+static const char controllerScript[] = {
 #include "DbgController.js.encoded"
```

Comment:
Good question. Didn't have an issue but let me force encode to Utf8 if it wasn't.